stage1/init/common/pod.go

```diff
@@ -276,6 +265,129 @@ func findHostPort(pm schema.PodManifest, name types.ACName) uint {
 	return port
 }
 
+// generateSysusers generates systemd sysusers files for a given app so that
+// corresponding entries in /etc/passwd and /etc/group are created in stage1.
+// This is needed to use the "User=" and "Group=" options in the systemd
+// service files of apps.
+// If there're several apps defining the same UIDs/GIDs, systemd will take care
+// of only generating one /etc/{passwd,group} entry
+func generateSysusers(p *stage1commontypes.Pod, ra *schema.RuntimeApp, uid_ int, gid_ int, uidRange *uid.UidRange) error {
+	var toShift []string
+
+	app := ra.App
+	appName := ra.Name
+
+	sysusersDir := path.Join(common.Stage1RootfsPath(p.Root), "usr/lib/sysusers.d")
+	toShift = append(toShift, sysusersDir)
```

Comment:
Hm... we call this function for each app so `Mkdir` makes the second app fail. Also, we're already doing the shift here so moving it to outside is not nice because we'll repeat the shifting code.

I think it's better to leave it as `MkdirAll`